src/com/epam/javacourse/homework4/Main.java

```diff
@@ -1,11 +1,14 @@
 package com.epam.javacourse.homework4;
 
+import java.nio.file.Path;
 import java.util.List;
 import java.util.Scanner;
 
 public class Main {
-    public static void main(String[] args) {
-        GreenhouseManagement greenhouse = new GreenhouseManagement();
+
+    public static void main(String[] args) throws InvalidChoiceException {
```

Comment:
This method throws not only `InvalidChoiceException`. Can we have the full list here to make it clear?